src/test/java/org/aksw/simba/squirrel/fetcher/utils/ZipArchiverTest.java

```diff
@@ -0,0 +1,24 @@
+package org.aksw.simba.squirrel.fetcher.utils;
+
+import static org.junit.Assert.assertNotNull;
+import static org.junit.Assert.fail;
+
+import java.io.File;
+import java.util.zip.ZipFile;
+
+import org.junit.Test;
+
+import com.mchange.util.AssertException;
+
+public class ZipArchiverTest {
+
+    @Test
+    public void testZipExtract() throws Exception{
+        String src = "C:\\Users\\kpten\\IdeaProjects\\Integration_Testing\\Squirrel\\src\\test\\resources\\sample_test.zip";
+        String dest = "C:\\Users\\kpten\\IdeaProjects\\Integration_Testing\\Squirrel\\src\\test\\resources\\extracted_files" ;
```

Comment:
Now, the test only works for you. :disappointed: 
So if you don't want to serve as the developer for this project for the rest of your live, you should make it more generic :wink: 
You can access resources using their name and `ZipArchiverTest.class.getClassLoader().getResource(name)` which returns a URL to the resource. Using this URL, you should typically be able to open a file (at least in a test class. This does not always work in other classes but that is a different story).

Reply:
@MichaelRoeder made the code more generic by using the above said getClassLoader(). Please have a look.